db/schema/transactionPerson.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Schema } from 'mongoose';
+
+import transaction from './transaction';
+import amountDetail from './amountDetail';
+import item from './item';
+
+const transactionPersonSchema: Schema = new mongoose.Schema({
+  id: String,
+  person: String,
+  transaction: transaction,
+  amountDetails: amountDetail,
```

Comment:
Should be `amountDetail` not amountDetails since the value is not an array of amountDetails but one singular amountDetail.